eng/common/pipelines/templates/steps/publish-1es-artifact.yml

```diff
@@ -23,7 +23,7 @@ steps:
     displayName: Set Failed Artifact Name
```

Comment:
See my other comment https://github.com/Azure/azure-sdk-tools/pull/7758#discussion_r1511884105 I think we can make this one step to set the artifact name.

Reply:
Ah excellent